wp-content/plugins/hurumap/src/IndicatorBlock/index.js

```diff
@@ -0,0 +1,39 @@
+// import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import Edit from './Edit';
+import Save from './Save';
+
+registerBlockType('hurumap/indicator-block', {
+  title: __('Indicator', 'hurumap'),
+  icon: 'admin-page', // https://developer.wordpress.org/resource/dashicons/#chart-pie
+  category: 'widgets',
+  attributes: {
+    title: {
+      type: 'string'
+    },
+    description: {
+      type: 'string'
+    },
+    sourceTitle: {
+      type: 'string'
+    },
+    sourceLink: {
+      type: 'string'
+    },
+    src: {
+      type: 'string'
+    },
+    srcId: {
+      type: 'number'
+    },
+    layout: {
+      type: 'string'
+    },
+    client: {
```

Comment:
Can you document a bit what are `src`, `srcId` and `client` (and how we use/why we need them)?